eahub/profiles/forms.py

```diff
@@ -20,7 +20,7 @@ def save(self, commit=True):
 
     class Meta(CaseInsensitiveUsernameFieldCreationForm.Meta):
         model = User
-        fields = ['email']
+        fields = ['name', 'email']
```

Comment:
Why this change? Does the signup flow still work properly after making it? I ask because this is a `ModelForm` and the `User` model doesn't have a `Name` field, so I'm not sure whether it's right or wrong.

Reply:
This was necessary to make sure the name appears before the email on the form.